services/core-web/src/components/mine/Incidents/MineIncident.js

```diff
@@ -125,6 +126,36 @@ export class MineIncident extends Component {
   handleSaveData = () => {
     const incidentExists = Boolean(this.props.formValues?.mine_incident_guid);
     const errors = Object.keys(flattenObject(this.props.formErrors));
+
+    let determinationTypeCode = null;
+    if (
+      this.props.formValues.determination_type_code &&
+      this.props.formValues.determination_type_code ===
+        Strings.INCIDENT_DETERMINATION_TYPES.dangerousOccurance
+    ) {
+      determinationTypeCode = true;
+    } else if (
+      this.props.formValues.determination_type_code &&
+      [
+        Strings.INCIDENT_DETERMINATION_TYPES.pending,
+        Strings.INCIDENT_DETERMINATION_TYPES.notADangerousOccurance,
+      ].includes(this.props.formValues.determination_type_code)
+    ) {
+      determinationTypeCode = false;
+    } else {
+      determinationTypeCode = this.props.formValues.mine_determination_type_code;
+    }
+
+    if (!this.props.formValues.status_code) {
+      if (!determinationTypeCode) {
+        this.props.formValues.status_code = "WNS";
```

Comment:
What are the possible statuses for an incident that is marked as a non dangerous occurrence? Are they always WNS and then Closed?

Reply:
That's correct., they are created as WNS and then go to the next stages which I guess are: Reviewing Severity Status/ Information Missing Status -> Under Review -> Closed.